dll/win32/devmgr/lang/pt-PT.rc

```diff
@@ -40,59 +40,59 @@ STRINGTABLE
+    IDS_DEV_CANT_SHARE_IRQ "Este dispositivo utiliza um recurso de Requisição de Interrupção (IRQ) que está em uso por outro dispositivo e não pode ser compartilhado.\nDeve alterar a configuração conflitante ou remover o controlador de modo real que está a causar o conflito."
     IDS_DEV_FAILED_ADD "Este dispositivo não está a funcionar correctamente porque %1 não está a funcionar correctamente."
-    IDS_DEV_DISABLED_SERVICE "O ReactOS não pode instalar os drivers para este dispositivo porque não consegue acessar o drive ou local de rede que possui os arquivos de configuração."
-    IDS_DEV_TRANSLATION_FAILED "Este dispositivo não está a responder ao seu driver."
+    IDS_DEV_DISABLED_SERVICE "O ReactOS não pode instalar os controladores para este dispositivo porque não consegue acessar o controlador ou local de rede que possui os arquivos de configuração."
+    IDS_DEV_TRANSLATION_FAILED "Este dispositivo não está a responder ao seu controlador."
     IDS_DEV_NO_SOFTCONFIG "O ReactOS não pode determinar as configurações para este dispositivo. Consulte a documentação que veio com este dispositivo e utilize a guia Recursos para definir a configuração."
     IDS_DEV_BIOS_TABLE "O firmware de seu sistema não possui informação suficiente para configurar e utilizar este dispositivo correctamente.\nPara utilizar este dispositivo, contacte o fabricante de seu computador para obter uma actualização de firmware ou BIOS."
     IDS_DEV_IRQ_TRANSLATION_FAILED "Este dispositivo requisita uma interrupção PCI mas está configurado para uma interrupção ISA (ou vice-versa).\nPor favor, utilize o sistema de configuração do computador para reconfigurar a interrupção para este dispositivo."
-    IDS_DEV_FAILED_DRIVER_ENTRY "O ReactOS não pode inicializar o driver do dispositivo para este hardware."
-    IDS_DEV_DRIVER_FAILED_PRIOR_UNLOAD "O ReactOS não pode carregar o driver do dispositivo para este hardware porque uma instância anterior do driver do dispositivo ainda está na memória."
-    IDS_DEV_DRIVER_FAILED_LOAD "O ReactOS não pode carregar o driver do dispositivo para este hardware. O driver pode estar corrompido ou ausente."
+    IDS_DEV_FAILED_DRIVER_ENTRY "O ReactOS não pode inicializar o controlador do dispositivo para este hardware."
+    IDS_DEV_DRIVER_FAILED_PRIOR_UNLOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware porque uma instância anterior do controlador do dispositivo ainda está na memória."
+    IDS_DEV_DRIVER_FAILED_LOAD "O ReactOS não pode carregar o controlador do dispositivo para este hardware. O controlador pode estar corrompido ou ausente."
     IDS_DEV_DRIVER_SERVICE_KEY_INVALID "O ReactOS não pode acessar este hardware porque suas informações sobre a chave do serviço não estão no Registo ou estão registadas incorrectamente."
```

Comment:
"incorretamente" ?

Reply:
incorreCtamente is Portuguese from Portugal